packages/modal/package.json

```diff
@@ -21,7 +21,8 @@
     "@hig/icon-button": "^1.0.1",
```

Comment:
I would bump up to match the lates versions of other `@hig` components. The current version of `@hig/icons` included isn't the themeable version.